args.md

```diff
@@ -302,13 +307,15 @@ structures for simulation or inference is therefore infeasible.
 
 ## ARG formats and `tskit`
 
-In classical ARGs, nodes often represent events (specifically, _common ancestor_,
-_recombination_, and _sampling_ events), with the genomic regions of inheritance
-encoded by storing a specific breakpoint location on each recombination node.
-In contrast, nodes in a `tskit` ARG correspond to _genomes_, and inherited regions
-are defined by intervals stored on *edges* (via the {attr}`~Edge.left` and 
-{attr}`~Edge.right` properties), rather than on nodes. Here, for example, is the
-edge table from our ARG:
+It is worth noting a subtle and somewhat philosophical
+difference between some classical ARG formulations, and the ARG formulation
+used in `tskit`. Clasically, nodes are taken to represent events
+(specifically, _common ancestor_, _recombination_, and _sampling_ events),
+and genomic regions of inheritance are encoded by storing a specific breakpoint location on
+each recombination node. In contrast, nodes in a `tskit` ARG correspond to _genomes_. More
```

Comment:
Should probably link to tskit definition of a node here, otherwise "genome" doesn't explain much.